latch_cli/docker_utils/__init__.py

```diff
@@ -309,14 +306,66 @@ def infer_commands(pkg_root: Path) -> List[DockerCmdBlock]:
     return commands
 
 
+def copy_file_commands(wf_type: WorkflowType) -> List[DockerCmdBlock]:
+    cmd = []
+
+    cmd += [
+        "",
+        "# Copy workflow data (use .dockerignore to skip files)",
+        "",
+        "copy . /root/",
+    ]
+
+    if wf_type == WorkflowType.snakemake:
+        cmd += [
+            "",
+            "# Latch snakemake workflow entrypoint",
+            "# DO NOT CHANGE",
+            "",
+            "copy .latch/snakemake_jit_entrypoint.py /root/snakemake_jit_entrypoint.py",
+        ]
+    elif wf_type == WorkflowType.nextflow:
+        cmd += [
+            "",
+            "# Latch nextflow workflow entrypoint",
+            "# DO NOT CHANGE",
+            "",
+            "run ln -s /root/.latch/bin/nextflow /root/nextflow",
+            "run ln -s /root/.latch/.nextflow /root/.nextflow",
+        ]
+
+    return cmd
+
+
+def generate_dockerignore(pkg_root: Path, *, wf_type: WorkflowType) -> None:
+    if wf_type != WorkflowType.nextflow:
+        return
+
+    dest = Path(pkg_root) / ".dockerignore"
+    if dest.exists() and not click.confirm(
+        f".dockerignore already exists at `{dest}`. Overwrite?"
+    ):
+        return
```

Comment:
throw an error if dest exists and is a file

Reply:
why throw an error instead of asking if user wants to overwrite the file? 
unless you meant exists and is directory?

Reply:
oh sorry yeah meant dir